### DISCUSSION TITLE
what are your favorite Git/GitHub courses ?

### DISCUSSION THREAD

**Discussion by vanesagab** [community/community#20211] [Other Feature Feedback, Questions, & Ideas] [answered]
What are your favorite tutorials/courses for Git & GitHub? 
Level: Beginner - Intermediate :octocat:

**Comment by armoredvortex** [comment]
Coding Train's Git and Github for Poets:
https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV

**Reply by vanesagab** [reply]
thanks! I'll check it.

**Comment by Tostisto** [comment]
https://lab.github.com/

**Comment by manasesjesus** [comment, marked as answer]
On September 1, 2022, GitHub Learning Lab will shut down. Instead, you can review [GitHub Skills](https://skills.github.com/).

The one mentioned by @ArmoredVortex is good 👍🏼 

There's one free course in Udemy that is also good: [Become an Expert in Git & GitHub in 4 Hours](https://www.udemy.com/course/git-expert-4-hours/)

**Reply by vanesagab** [reply]
thanks
